plugins/scaffolder-backend/src/service/router.test.ts

```diff
@@ -230,14 +235,12 @@ describe('createRouter', () => {
 
       jest
         .spyOn(permissionApi, 'authorizeConditional')
-        .mockImplementation(async () => [
-          {
-            result: AuthorizeResult.ALLOW,
-          },
-          {
-            result: AuthorizeResult.ALLOW,
-          },
-        ]);
+        .mockImplementation(async permissionQueries => {
```

Comment:
comment: these changes to the permissionApi mocks will likely conflict with #25969